Containerfile

```diff
@@ -4,17 +4,11 @@ MAINTAINER LSIT Systems <lsitops@lsit.ucsb.edu>
 
 USER root
 
-RUN sudo apt update ; apt install -y cargo rustc; apt-get clean
+RUN sudo apt update ; apt-get clean
 
-RUN mamba install -c conda-forge -y ipympl jupyter_bokeh mplcursors nltk nodejs pytest spacy tweepy
+RUN pip install PTable pytest pandas beautifulsoup4 numpy gensim scikit-learn matplotlib nltk arpa morfessor
```

Comment:
We should be able to remove at least pandas, numpy, and matplotlib from this list.  I would keep them in the Jenkinsfile test for good measure, though.  I don't know if keeping it would harm anything or not but pip takes more liberties with versions than conda/mambda does and these are provided by jupyter upstream now.